rs/pocket_ic_server/src/pocket_ic.rs

```diff
@@ -1699,6 +1706,206 @@ impl PocketIcSubnets {
+            //   };
+            //   is_production = opt true;
+            //   enable_dapps_explorer = opt false;
+            //   assigned_user_number_range = opt record {
+            //     10_000 : nat64;
+            //     7_569_744 : nat64;
+            //   };
+            //   new_flow_origins = opt vec { "https://id.ai" };
+            //   archive_config = opt record {
+            //     polling_interval_ns = 15_000_000_000 : nat64;
+            //     entries_buffer_limit = 10_000 : nat64;
+            //     module_hash = blob "\17\4d\7e\2c\3d\4d\3b\0f\34\61\63\4d\49\ea\c2\85\55\5a\97\48\de\7c\c7\a8\53\cf\ea\00\41\52\a1\97";
+            //     entries_fetch_limit = 1_000 : nat16;
+            //   };
+            //   canister_creation_cycles_cost = opt (0 : nat64);
+            //   analytics_config = opt opt variant {
+            //     Plausible = record {
+            //       domain = opt "identity.internetcomputer.org";
+            //       track_localhost = null;
+            //       hash_mode = null;
+            //       api_host = null;
+            //     }
+            //   };
+            //   related_origins = opt vec {
+            //     "https://id.ai";
+            //     "https://identity.ic0.app";
+            //     "https://identity.internetcomputer.org";
+            //     "https://identity.icp0.io";
+            //   };
+            //   feature_flag_continue_from_another_device = null;
+            //   captcha_config = opt record {
+            //     max_unsolved_captchas = 500 : nat64;
+            //     captcha_trigger = variant { Static = variant { CaptchaDisabled } };
+            //   };
+            //   dummy_auth = opt null;
+            //   register_rate_limit = opt record {
+            //     max_tokens = 25_000 : nat64;
+            //     time_per_token_ns = 1_000_000_000 : nat64;
+            //   };
+            // },
+            let internet_identity_test_args = InternetIdentityInit {
+                assigned_user_number_range: None, // DIFFERENT FROM ICP MAINNET
+                archive_config: None,             // DIFFERENT FROM ICP MAINNET
+                canister_creation_cycles_cost: Some(0),
+                register_rate_limit: Some(RateLimitConfig {
+                    max_tokens: 25_000,
+                    time_per_token_ns: 1_000_000_000,
+                }),
+                captcha_config: Some(CaptchaConfig {
+                    max_unsolved_captchas: 500,
+                    captcha_trigger: CaptchaTrigger::Static(StaticCaptchaTrigger::CaptchaDisabled),
+                }),
+                related_origins: None,  // DIFFERENT FROM ICP MAINNET
+                new_flow_origins: None, // DIFFERENT FROM ICP MAINNET
+                openid_google: Some(Some(OpenIdConfig {
+                    client_id:
+                        "775077467414-rgoesk3egruq26c61s6ta8bpjetjqvgo.apps.googleusercontent.com"
```

Comment:
Could you use `775077467414-q1ajffledt8bjj82p2rl5a09co8cf4rf.apps.googleusercontent.com` instead?

The one you are using is the production one and we don't want to use it for test environments

Reply:
Fixed: https://github.com/dfinity/ic/pull/6288/commits/c1def96b379818bc229e336f63749c9421de7e61